ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -667,6 +667,10 @@ ethernet_interfaces:
     isis_passive: < boolean >
     isis_metric: < integer >
     isis_network_point_to_point: < boolean >
+    priority_flow_control: < true | false >
+    priority_flow_control_priorities:
+      - priority : < 0-7 >
```

Comment:
Please update data model to make priority the key value, since it's unique.
```yaml
    priority_flow_control: < true | false >
    priority_flow_control_priorities:
      < 0-7 >:
        enable_pfc: < true | false >
```